ambry-quota/src/main/java/com/github/ambry/quota/storage/MySqlStorageUsageRefresher.java

```diff
@@ -359,6 +366,32 @@ static long secondsToNextMonthTick(String currentMonthValue, long offsetInSecond
     return secondsNextMonth - time.seconds();
   }
 
+  /**
+   * Convert an {@link AggregatedAccountStorageStats} to a map from account id to container id to storage usage.
+   * The account id and container id are keys of the outer map and the inner map, they are both in string format.
+   * @param aggregatedAccountStorageStats The {@link AggregatedAccountStorageStats}.
+   * @param usePhysicalStorageUsage True to use physical storage, false to use logical storage usage.
+   * @return A map from account id to container id to storage usage.
+   */
+  static Map<String, Map<String, Long>> convertAggregatedAccountStorageStatsToMap(
```

Comment:
Why did you move this method out of StorageStatsUtil? Seems that StorageStatsUtil.java is empty now and we can remove it?

Reply:
removed